src/systemctl/systemctl.c

```diff
@@ -268,20 +268,13 @@ static void ask_password_agent_open_if_enabled(void) {
         ask_password_agent_open();
 }
 
-static void polkit_agent_open_if_enabled(void) {
-
+static void polkit_agent_open_maybe(void) {
         /* Open the polkit agent as a child process if necessary */
 
-        if (!arg_ask_password)
-                return;
-
         if (arg_scope != UNIT_FILE_SYSTEM)
                 return;
 
```

Comment:
Just dumping my thoughts, does it make sense to check other scopes like global... ?

Reply:
No. Global scope is done through filesystem operations, not over bus, and pid1 is not involved in this at all.